### ISSUE TITLE
Develop geographic-rule-based methods for detecting duplicate requests

### ISSUE THREAD

**Issue by ryanmswan** [hackforla/311-data#239]
### Overview
Add unique ID column representing rows that are identified as duplicate reports of the same issue based on latitude, longitude, and date opened.

### Action Items
- [ ] based on time period and geographic location, develop rules that allow for duplicate submissions to be identified
- [ ] add location ID column

**Comment by johnr54321** [comment]
This ticket is a "nice to have" for prototype.  We are waiting on Genna.
